src/core/logging.py

```diff
@@ -3,12 +3,11 @@
 import sys
 
 import structlog
-from dependency_injector.wiring import Provide
 
-from src.settings import Container, Settings
+from src.settings import Settings
 
 
-def init_logging(settings: Settings = Provide[Container.settings], name: str = "log.log"):
+def init_logging(settings: Settings, name: str = "log.log"):
```

Comment:
Кажется name больше не нужно.
```suggestion
def init_logging(settings: Settings):
```